crates/cxx-qt-gen/src/lib.rs

```diff
@@ -20,6 +21,7 @@ pub use generator::{
     GeneratedOpt,
 };
 pub use parser::Parser;
+pub use preprocessor::self_inlining::qualify_self_types;
```

Comment:
I'm not too happy with the name `preprocessor` here, as it's reminiscent of the C preprocessor, which does something very different and is invoked before the parser, not after.

Why not just move the `qualify_self_types` function into the `naming` module and then just call `naming::qualify_self_types`?

Side-note: In Rust it's best-practice to only `use` the module that a function is in and then call the function with `module_name::function_name`, so please stick to that convention. (I know we haven't done this ourselves consistently before :sweat_smile: :see_no_evil: )